components/kyma-metrics-collector/pkg/process/event_stream_test.go

```diff
@@ -30,7 +30,7 @@ func TestParse(t *testing.T) {
 		{
 			name: "with Azure, 2 vm types, 3 pvcs(5,10 and 20Gi) and 2 svcs(1 clusterIP and 1 LoadBalancer)",
 			input: Input{
-				shoot:    kmctesting.GetShoot("testShoot", kmctesting.WithAzureProviderAndStandardD8V3VMs),
+				provider: Azure,
```

Comment:
IMO, we should use the value as it would appread in the KEB response, not the same const which we are testing.

Reply:
it is already lowercased [here](https://github.com/muralov/control-plane/blob/7149f36d23ef135876202488436debdbbeed57d5/components/kyma-metrics-collector/pkg/process/process.go#L360) and it has lowercase value in `input`